pkg/apis/pipeline/v1alpha1/pipeline_validation_test.go

```diff
@@ -24,6 +24,73 @@ import (
 	tb "github.com/tektoncd/pipeline/test/builder"
 )
 
+func TestPipeline_Validate_Error(t *testing.T) {
+	tests := []struct {
+		name string
+		p    *v1alpha1.Pipeline
+	}{
```

Comment:
Let's collapse these brackets to preserve horizontal space:

```
tests := []struct {
  ...fields...
}}{
  ...case...
}, {
  ...case...
}}
```

Reply:
Addressed in most recent commit (and also changed for the other tests in that file) - let me know how it looks!